VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionCommentsFragment.java

```diff
@@ -41,6 +44,9 @@
     @Inject
     Context context;
 
+    @InjectExtra(Router.EXTRA_DISCUSSION_TOPIC_IS_CLOSED)
+    boolean isTopicClosed;
```

Comment:
This should be private I guess, just like the `discussionComment` injection. The other variables should probably also be private.